### DISCUSSION TITLE
Login screen code file

### DISCUSSION THREAD

**Discussion by pcmodder2001** [jellyfin/jellyfin#9031] [General]
Hiya, 

im trying to find out where the login code file is saved in the jellyfin directory. 

im trying to add another button or maybe 2. ive tried to use the login disclaimer in general settings but that does not apply the same css as the other buttons. ive tried to manually add the css in but also does not work so wanting to try get it in the achuall file but not no look in finding it.

**Comment by pypp** [comment]
I think you are refering to [this](https://github.com/jellyfin/jellyfin-web/blob/master/src/controllers/session/login/index.html).
